tools/azure-plugin/generator/TestProjects/Local/Unbranded-TypeSpec/src/UnbrandedTypeSpec.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the UnbrandedTypeSpec client library for developing .NET applications with rich experience.</Description>
+    <AssemblyTitle>SDK Code Generation UnbrandedTypeSpec</AssemblyTitle>
+    <Version>1.0.0-beta.1</Version>
+    <PackageTags>UnbrandedTypeSpec</PackageTags>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
This should use the standard eng sys definition for target frameworks.   The goal is, after all, to generate Azure SDK clients from the Azure plugin.

Reply:
same comment as https://github.com/Azure/azure-sdk-for-net/pull/45617#discussion_r1729210586

Reply:
Please see:  [this comment](https://github.com/Azure/azure-sdk-for-net/pull/45617#discussion_r1729296230)